docs/plugins/contributors/fetchedUsers.json

```diff
@@ -239,21 +239,6 @@
     "position": "",
     "github": "https://github.com/infi-pc"
   },
-  {
-    "id": 2528846,
```

Comment:
Why is this contributor being removed? Is it expected?

Reply:
yep, not manually, the script removed him 🤷🏻